server/ansible/roles/pbench-server-systemd-service/vars/main.yml

```diff
@@ -1,6 +1,2 @@
 ---
-pbench_server_service_file_suffix: "{{ 'rhel7' if ansible_distribution_major_version == '7' and 
-                                                  (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS') 
-                                       else 'default' }}"
-
-pbench_server_service_file:  /opt/pbench-server/lib/systemd/pbench-server.service.{{ pbench_server_service_file_suffix }}
+pbench_server_service_file:  /opt/pbench-server/lib/systemd/pbench-server.service
```

Comment:
This change is in conflict with the the PR https://github.com/distributed-system-analysis/pbench/pull/1873,  since this a service file related change I addressed it in the PR otherwise these 2 PR will become defendant on each other

Reply:
We'll land #1873 first, so when #1874 is rebased this change will disappear from this commit.

Reply:
Reverted.